src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -322,4 +323,17 @@ class InstallAppsTask extends AbstractServerTask {
         }
         return false
     }
+
+    void createApplicationFolder(String appDir) {
+        File serverDir = getServerDir(project)
+        File applicationDirectory = new File(serverDir, appDir)
+
+        try {
+            if (!applicationDirectory.exists()) {
+                applicationDirectory.mkdir()
+            }
+        } catch (Exception e) {
+            throw new GradleException('There was a problem creating the apps or dropins folder in the server directory.', e)
```

Comment:
I think you could use the actual applicationDirectory path in this message instead of saying "apps or dropins".